moabb/pipelines/utils.py

```diff
@@ -50,6 +50,31 @@ def create_pipeline_from_config(config):
     return pipeline
 
 
+def parse_Deeplearning_callbacks(config):
```

Comment:
We usually follow the convention of camel for classes and snake_case for functions. 
```suggestion
def parse_deep_callbacks(config):
```